src/core/network.js

```diff
@@ -75,6 +75,21 @@ var NetworkManager = (function NetworkManagerClosure() {
     return array.buffer;
   }
 
+//#if !(CHROME || FIREFOX || MOZCENTRAL)
+  var supportsMozChunked = (function supportsMozChunkedClosure() {
+    var x = new XMLHttpRequest();
+    // Firefox requires that .open() is called before setting responseType.
+    // https://bugzilla.mozilla.org/show_bug.cgi?id=707484
+    x.open('GET', 'http://example.com');
```

Comment:
Is it possible to move this line into the `try` block instead.

The reason that I'm asking is that when testing the preview (i.e. the `GENERIC` build), Firefox threw an error on this line and the viewer thus failed to load the PDF file. I tracked this down to the fact that I'm using the NoScript addon, which blocked this request (since I hadn't white-listed that particular domain).

Considering that NoScript is a fairly common addon, it's conceivable that more people could run into this. In this case, it thus seems better to me that streaming gets disabled, than the entire viewer breaking.